app/models/transaction.rb

```diff
@@ -148,15 +155,30 @@ def set_date
     self.date = Time.current
   end
 
+  def calculate_amount
+    self.amount = Money.new(amount_cents - comission.to_d * 100, currency)
+    add_comission_to_comment
```

Comment:
@yunixon I think it will be better to call this method with `!` symbol, cos it makes updates in the class, so it should looks like: `self.add_comission_to_comment!`